.ci/setup.sh

```diff
@@ -37,17 +38,17 @@ else
 	die "Unsupported architecture: $arch"
 fi
 
-echo "Install Qemu"
-bash -f ${cidir}/install_qemu.sh
-
-echo "Install shim"
-bash -f ${cidir}/install_shim.sh
-
-echo "Install proxy"
-bash -f ${cidir}/install_proxy.sh
+if [ "${INSTALL_KATA}" == "TRUE" ];then
+	echo "Install Kata from sources"
+	bash -f ${cidir}/install_kata_from_source.sh
+fi
 
-echo "Install runtime"
-bash -f ${cidir}/install_runtime.sh
+# Install docker after install kata image
+# install_kata_from_source.sh: Creates the image from source
+# and requires runc as default runtime
```

Comment:
To create the image, we need docker, right? Then we need to install docker first and then call `install_kata_from_source.sh` ?

Reply:
-_- ups.